packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/job-dashboard.html

```diff
@@ -15,6 +15,14 @@
                     ></pci-project-data-processing-job-status>
                 </oui-tile-description>
             </oui-tile-definition>
+            <!-- Tile item :: Return Code -->
+            <oui-tile-definition
+                data-term="{{ :: 'data_processing_details_information_return_code_label' | translate }}"
+                title="{{ ::'data_processing_details_information_return_code_label' | translate }}"
+                data-description="{{ $ctrl.job.returnCode }}"
+                data-ng-if="$ctrl.job.returnCode !== undefined"
```

Comment:
```suggestion
                data-ng-if="$ctrl.job.returnCode"
```
I think we don't need "!== undefined"

Reply:
Good point

Reply:
@ghyenne Actually, on second thought, that would be an issue. returnCode is expected to be an int and can be 0 if all went well. If I use data-ng-if="$ctrl.job.returnCode" then it would hide the field when I want to display 0.

Reply:
Indeed 0 is interpreted as false.